R/cleaning_drug_exposure_extract_from_bigQueryDB.R

```diff
@@ -0,0 +1,32 @@
+# Load necessary library
+library(readr)

```

Comment:
Should we also include at the beginning the code for how the data was extracted from the BigQuery DB?